src/content/community/releases/_index.en.md

```diff
@@ -5,6 +5,11 @@ weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
 
+## v0.15.5 (July 23, 2024)
```

Comment:
```suggestion
## v0.15.5 (July 23, 2024), replacing v0.15.4
```

to explain why the latter doesn’t have a changelog entry.

Reply:
For 0.16, we had 3 "failed" releases, 0.16.4-6, which aren't listed, so we have v0.16.3 ... v0.16.7. I think we can do the same here and not mention v0.15.4 since it's not a release we want to advertise.

Reply:
I understand we don’t want to advertise it, but I still find it odd to have a changelog that skips versions — it makes me wonder what was in those versions. There’s a `v0.16.1/v0.16.2` entry further down...

Reply:
yeah I did that for v0.16.1 which was also a broken release but I don't think the entry conveys that, ie we don't want anyone installing 0.16.1. It might appear to some that 0.16.2 just has the same release notes as 0.16.1. Same case with 0.15.4. I don't view this file as a strict, continuous changelog, but to advertise the release notes for working versions. 0.15.4 does not fit that criteria, ie we don't want anyone downloading it.

Reply:
> it makes me wonder what was in those versions

I added a note here: https://github.com/submariner-io/releases/releases/tag/v0.15.4

Reply:
OK, so perhaps the best solution is to actually delete the release, at least the artifacts — what do you think?

Reply:
Perhaps. I'll also add a similar note here about 0.15.4